templates/react/party/server.ts

```diff
@@ -0,0 +1,50 @@
+import type * as Party from "partykit/server";
+
+export default class Server implements Party.Server {
+  count = 0;
+
+  constructor(readonly room: Party.Room) {}
+
+  onConnect(conn: Party.Connection, ctx: Party.ConnectionContext) {
+    // A websocket just connected!
+    console.log(
+      `Connected:
+  id: ${conn.id}
+  room: ${this.room.id}
+  url: ${new URL(ctx.request.url).pathname}`
+    );
+
+    // send the current count to the new client
+    conn.send(this.count.toString());
+  }
+
+  onMessage(message: string, sender: Party.Connection) {
+    // let's log the message
+    console.log(`connection ${sender.id} sent message: ${message}`);
+    // we could use a more sophisticated protocol here, such as JSON
+    // in the message data, but for simplicity we just use a string
+    if (message.trim() === "increment") {
```

Comment:
```suggestion
    if (message === "increment") {
```
let's be explicit with messages